bin/nanocld/src/services/process/mod.rs

```diff
@@ -34,6 +34,7 @@ pub fn ntex_config(config: &mut web::ServiceConfig) {
   config.service(wait_processes);
   config.service(stats_processes);
   config.service(count_processes);
+  config.service(processes_stats_by_name);
```

Comment:
You have to add a test, even a small one to not lose coverage

Reply:
Done